apps/example-nest/src/app/post.controller.spec.ts

```diff
@@ -76,16 +76,21 @@ describe('PostController', () => {
         .query('skip=0')
         .expect(400)
         .expect({
-          issues: [
-            {
-              code: 'invalid_type',
-              expected: 'string',
-              message: 'Required',
-              path: ['take'],
-              received: 'undefined',
-            },
-          ],
-          name: 'ZodError',
+          queryResult: {
```

Comment:
Do you think it would be better if we kept the `post.controller.ts` and `post.controller.spec.ts` files as-is to make sure we don't have any regressions and create a separate controller and respective test file for the new handlers?